lib/render/editor/editor.html

```diff
@@ -161,7 +161,7 @@
       },
 
       _setInitialXml: function (blockly) {
-        const editorDefaultContent = $("#mu-custom-editor-default-value")[0];
+        const editorDefaultContent = $("#default_content")[0];
```

Comment:
Why are we using an internal class here?

Reply:
not really sure what you mean by internal class, would you elaborate?

Reply:
I mean a class that is not prefixed with `mu`